src/signals/incident/components/form/ContainerSelect/Selector/WfsLayer/WfsLayer.tsx

```diff
@@ -27,17 +26,22 @@ const WfsLayer: FunctionComponent<WfsLayerProps> = ({ children, zoomLevel = {} }
 
   const getBbox = (map: MapType): string => {
     const bounds = map.getBounds();
-    const bbox = `${bounds.getWest()},${bounds.getSouth()},${bounds.getEast()},${bounds.getNorth()},${SRS_NAME}`;
-    return `&${L.Util.getParamString({
-      bbox,
-    }).substring(1)}`;
+
+    return `<BBOX><PropertyName>geometrie</PropertyName><gml:Envelope srsName="${SRS_NAME}"><lowerCorner>${bounds.getWest()} ${bounds.getSouth()}</lowerCorner><upperCorner>${bounds.getEast()} ${bounds.getNorth()}</upperCorner></gml:Envelope></BBOX>`;
   };
 
   const [bbox, setBbox] = useState('');
   const [data, setData] = useState<FeatureCollection>(NO_DATA);
 
+  const wfsFilter = useMemo<string>(
+    () => `&Filter=<Filter><And>${meta.wfsFilters?.map(filter => filter.value)}${bbox}</And></Filter>`,
```

Comment:
When you create the filter, add the `Add` operator only when the filter is present:
```suggestion
() =>  `&Filter=<Filter>{ meta.wfsFilter ? `<And>${meta.wfsFilter>${bbox}</And>` : bbox } </Filter>`,
```